app/config/locale/translations/gu.json

```diff
@@ -3,31 +3,31 @@
     "settings.locale": "gu",
     "settings.direction": "ltr",
     "emails.sender": "%s ટીમ",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
-    "locale.country.unknown": "अज्ञात",
+    "emails.verification.subject": "ખાતાની ચકાસણી",
+    "emails.verification.hello": "નમસ્કાર {{name}}",
+    "emails.verification.body": "તમારું ઇમેઇલ સરનામું ચકાસવા માટે આ લિંકને અનુસરો.",
+    "emails.verification.footer": "જો તમે આ સરનામાંની ચકાસણી કરવાનું ન કહ્યું હોય, તો તમે આ સંદેશને અવગણી શકો છો.",
+    "emails.verification.thanks": "આભાર",
+    "emails.verification.signature": "{{project}} ટીમ",
+    "emails.magicSession.subject": "પ્રવેશ કરો",
+    "emails.magicSession.hello": "નમસ્કાર,",
+    "emails.magicSession.body": "પ્રવેશ કરવા માટે આ લિંકને અનુસરો.",
+    "emails.magicSession.footer": "જો તમે આ ઇમેઇલનો ઉપયોગ કરીને પ્રવેશ કરવાનું ન કહ્યું હોય, તો તમે આ સંદેશને અવગણી શકો છો.",
+    "emails.magicSession.thanks": "આભાર",
+    "emails.magicSession.signature": "{{project}} ટીમ",
+    "emails.recovery.subject": "પાસવર્ડ ફરીથી સેટ કરો",
+    "emails.recovery.hello": "નમસ્કાર {{name}}",
+    "emails.recovery.body": "તમારો {{project}} પાસવર્ડ ફરીથી સેટ કરવા માટે આ લિંકને અનુસરો.",
+    "emails.recovery.footer": "જો તમે તમારો પાસવર્ડ ફરીથી સેટ કરવાનું ન કહ્યું હોય, તો તમે આ સંદેશને અવગણી શકો છો.",
+    "emails.recovery.thanks": "આભાર",
+    "emails.recovery.signature": "{{project}} ટીમ",
+    "emails.invitation.subject": "%S ટીમને %s પર આમંત્રણ",
```

Comment:
```suggestion
    "emails.invitation.subject": "%s ટીમને %s પર આમંત્રણ",
```

Reply:
Oh I'm so sorry.. will change it right now.